### PR TITLE
Update Django tutorial 3-4: extends tag

### PR DESCRIPTION
**Pull request by MarissaB**
https://github.com/MicrosoftDocs/visualstudio-docs/issues/8864

### CODE REVIEW

**Comment by MarissaB** [comment]
@microsoft-github-policy-service agree [company=""]

**Comment by MarissaB** [comment]
whoops, let's try this one:

@microsoft-github-policy-service agree

**Comment by Jak-MS** [comment]
@cwebster-99 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/python/learn-django-in-visual-studio-step-03-serve-static-files-and-add-pages.md [hunk 1]
```diff
@@ -272,29 +272,34 @@ The following steps demonstrate inheritance:
...
     {% endblock %}
     ```
 
-1. Modify the *templates/HelloDjangoApp/about.html* file to refer to the base template and override the content block:
-
+3. Modify the *templates/HelloDjangoApp/about.html* file in the same way to make the layout template available. Add the same line from step 1 in the HTML page (above the html tag):
     ```html
     {% extends "HelloDjangoApp/layout.html" %}
+    ```
+4. Then using inheritance, implement the template within the body tag to override the content block:
```
Comment (cwebster-99) [inline docs/python/learn-django-in-visual-studio-step-03-serve-static-files-and-add-pages.md:291]:
```suggestion
4. Then, using inheritance, implement the template within the body tag to override the content block:
```

**Review by cwebster-99 — APPROVED** [review]

**Comment by cwebster-99** [comment]
#sign-off

**Review by cwebster-99 — APPROVED** [review]

**Comment by cwebster-99** [comment]
#sign-off

**Pull request merged**